### ISSUE TITLE
Use of abstract class with open val interface will result in a false positive

### ISSUE THREAD

**Issue by matejdro** [skydoves/compose-stability-analyzer#178]
**Please complete the following information:**
- Library Version: Any version from 0.9.0 onwards. 0.8.0 does not exhibit this issue

 
**Describe the Bug:**

Analyze the following code with the compose analyzer gradle plugin:

```kotlin
@Composable
fun Greeting(testClass: TestClass, modifier: Modifier = Modifier) {
    Text(
        text = "Hello ${testClass.input}!",
        modifier = modifier
    )
}

data class TestClass(val input: String) : AbstractClass()


abstract class AbstractClass {
    open val testInterface: TestInterface?
        get() = null
}


interface TestInterface
```

**Expected Behavior:**

Compose's inference says that this class is stable, so CSA should not complain:

```
restartable skippable scheme("[androidx.compose.ui.UiComposable]") fun com.example.stabilitycheckbreakdemo.Greeting(
  stable testClass: TestClass
  stable modifier: Modifier? = @static <expression>
)
```

**Actual Behavior:**

Stability analyzer will detect that class as having RUNTIME stability:

```
> ❌ Stability check failed!
  
  The following composables have changed stability:
  
  + com.example.stabilitycheckbreakdemo.Greeting (new composable):
      testClass: RUNTIME
      modifier: STABLE
```

**Reproducing project**

For a simpler reproduction, here is a ready made project:

[Archive.zip](https://github.com/user-attachments/files/29623881/Archive.zip). 

Just extract it and run `./gradlew debugStabilityCheck`
